ci/ios/create-vm/scripts/install-brew-dependencies.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# shellcheck source=/dev/null
+source ~/.bash_profile
+
+if which brew 
```

Comment:
I would recommend a safer alternative to check if a program is in the user's `PATH`:

```bash
if command -v brew &>/dev/null; then
    # `brew` is in PATH
fi
```

https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script/677212#677212